yb-voyager/src/tgtdb/suites/yugabytedbSuite.go

```diff
@@ -189,6 +189,20 @@ var YBValueConverterSuite = map[string]ConverterFn{
 			return columnValue, nil
 		}
 	},
+	"io.debezium.data.Json": func(columnValue string, formatIfRequired bool) (string, error) {
```

Comment:
How about enhancing test automation to also have json and enum values?

Reply:
Its already there in PG tests and the issue caught in CDC but we don't have live migration datatypes automation for that yet.